### PR TITLE
Add delete-network-filter-via-wfp-api.yml and enumerate-network-filters-via-wfp-api.yml

### PR DESCRIPTION
**Pull request by jtothej**
Add two new CAPA rules:
host-interaction/network/traffic/filter/delete-network-filter-via-wfp-api.yml
host-interaction/network/traffic/filter/enumerate-network-filters-via-wfp-api.yml

### CODE REVIEW

File: host-interaction/network/traffic/filter/enumerate-network-filters-via-wfp-api.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+rule:
+  meta:
+    name: enumerate network filters via WFP API
+    namespace: host-interaction/network/traffic/filter
+    authors:
+      - jakub.jozwiak@mandiant.com
+    scopes:
+      static: function
+      dynamic: call
```
Comment (williballenthin) [inline host-interaction/network/traffic/filter/enumerate-network-filters-via-wfp-api.yml:9]:
```suggestion
      dynamic: thread
```

File: host-interaction/network/traffic/filter/enumerate-network-filters-via-wfp-api.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    att&ck:
```
Comment (williballenthin) [inline host-interaction/network/traffic/filter/enumerate-network-filters-via-wfp-api.yml:10]: maybe remove these?

**Review by williballenthin — APPROVED** [review]
thank you!

File: host-interaction/network/traffic/filter/enumerate-network-filters-via-wfp-api.yml (hunk 1 above)
Comment (jtothej) [reply host-interaction/network/traffic/filter/enumerate-network-filters-via-wfp-api.yml]: Thanks! Updated in https://github.com/mandiant/capa-rules/pull/930/commits/afafccaf8f177cefba9eb4475f294a839ec5ca1a

File: host-interaction/network/traffic/filter/enumerate-network-filters-via-wfp-api.yml (hunk 2 above)
Comment (jtothej) [reply host-interaction/network/traffic/filter/enumerate-network-filters-via-wfp-api.yml]: Thanks! Updated in https://github.com/mandiant/capa-rules/pull/930/commits/afafccaf8f177cefba9eb4475f294a839ec5ca1a

**Review by mr-tz — APPROVED** [review]
thanks!

**Pull request merged**